src/connectors/supla.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const filter = new MetadataFilter({
+	artist: cleanUpArtist,
+});
+
+Connector.playerSelector = 'main';
+
+Connector.artistTrackSelector = '.RadioHeader__WidgetContent-sc-17ofob1-4.irUZCR';
+
+Connector.playButtonSelector = '.r-play-button.r-toggle-playstate-button.r-touch-button';
```

Comment:
It's better to avoid multiple classes in a selector. Using `.r-play-button` only is enough.